### ISSUE TITLE
Duplicate classes error on Android

### ISSUE THREAD

**Issue by jcraane** [korlibs/klock#49]
We have some problems integrating Klock version 1.4.0 into our Kotlin multiplatform project. We must be doing something wrong but cannot pinpoint this at the moment. Our project structure is as follows (see als https://github.com/jcraane/KotlinNativeStarter) with the following dependencies:

```
Root project
    'settings.gradle'
        enableFeaturePreview("GRADLE_METADATA")
    common (our common code)
        'build.gradle'
            commonMain -> api "com.soywiz.korlibs.klock:klock:$klockVersion"
            iosMain -> implementation("com.soywiz:klock-iosx64:$klockVersion")
    android
        app (Android module)
            'build.gradle'
                implementation("com.soywiz:klock-locale-android:1.4.0")
                implementation("com.soywiz:klock-android:1.4.0")
    iosApp
```

We use the DateTime classes in the common, IOS and Android project. Compilation works fine with the above settings but if we try to run the Android application we get the following error (for all duplicate classes:

`java.lang.RuntimeException: java.lang.RuntimeException: Duplicate class com.soywiz.klock.BuildConfig found in modules classes.jar (com.soywiz.korlibs.klock:klock-android-debug:1.4.0) and classes.jar (com.soywiz:klock-android:1.4.0)`

Is there something wrong with our configuration?

Thanks!

**Comment by jcraane** [comment]
I managed to work-around the issue by excluding the klock-android-debug artefact from the all configurations in the Android project by adding the following to the build.gradle.kts of the app module:

```
configurations.all {
        exclude(
            group = "com.soywiz.korlibs.klock", module = "klock-android-debug"
        )
    }
```

With this I am able to run the Android app but when I try to compile the IOS app it outputs the following error:

```
e: Compilation failed: Could not find serialized descriptor for index: 6687674674430108232 com.soywiz.klock,DateTimeTz,minus

 * Source files: Api.kt, ApiResponse.kt, RealApi.kt, Person.kt, Sample.kt, SampleIos.kt
 * Compiler version info: Konan: 1.2.1 / Kotlin: 1.3.31
 * Output kind: FRAMEWORK

e: java.lang.IllegalStateException: Could not find serialized descriptor for index: 6687674674430108232 com.soywiz.klock,DateTimeTz,minus
```

Is this a know issue or is there something else wrong with my configuration?

**Comment by jcraane** [comment]
This issue is not relevant anymore now Klock 1.5.0 is released.

**Comment by mendess** [comment]
@jcraane I'm having this same issue and I can't find a way to fix it. Can you point me to the commit in this repo that fixes is or some other guide? Thanks!

**Comment by jcraane** [comment]
Mmm, I don't know the exact commit that it was. I think it was solved by just using 1.5. Is it the same error as above? Perhaps not really a short-term solution but in most of my KMM projects we migrated to: https://github.com/Kotlin/kotlinx-datetime

**Comment by mendess** [comment]
@jcraane I meant that I'm getting this same error in my library, no klock, so I wanted to know what I could do to fix it
